### ISSUE TITLE
Pytorch version

### ISSUE THREAD

**Issue by MDanialKhan** [chaimeleon-eu/CHAIMELEON-Platform#22]
I have a question regarding the pytorch version being used when we are running scripts on jobman. The current version is 2.6.0, of the dev branch when the current stable release is [2.5.1](https://pytorch.org/get-started/locally/).

 
![image](https://github.com/user-attachments/assets/a76aa152-c32a-411c-99d3-4ee48cd762a4)

I have attached this image for your reference.

The updates break our code as we are using a previous stable version of pytorch. Is there a way to restrict the version ourselves? as we are not overriding the version that is used by the jobman images.

**Comment by Palollo** [comment]
That's true, the "nightly" build is used in case of the image with CUDA, as you can see here:
https://github.com/chaimeleon-eu/workstation-images/blob/bea80dbdd5cdfd3ceddb1a53b1da11f7b343fac1/ubuntu-python-pytorch/Dockerfile#L39

That's probably due to compatibility reasons with CUDA 11.8 when that dockerfile was created. I think this because on the other hand the image for CPU (without CUDA) is built with the stable version of pytorch (as seen in the line before).
Maybe now (after a year) there is no problem to use the stable version also with CUDA libraries.

So a new version of the image with cuda and pytorch stable (2.5.1) has been uploaded and you can try with:
`jobman -i ubuntu-python-pytorch:3.10cuda11 ...`
or:
`jobman -i ubuntu-python-pytorch:latest-cuda ...`
